config/nav.yml

```diff
@@ -190,7 +190,9 @@ nav:
             - Available Broker types: eventing/brokers/broker-types/README.md
             # add default IMC broker page, page explaining broker types
             - Channel based Broker: eventing/brokers/broker-types/channel-based-broker/README.md
-            - Apache Kafka: eventing/brokers/broker-types/kafka-broker/README.md
+            - Apache Kafka: 
+              - About Apache Kafka Broker: eventing/brokers/broker-types/kafka-broker/README.md
+              - Configuring Kafka features: eventing/brokers/broker-types/kafka-broker/configuring-kafka-features.md
```

Comment:
I wonder if this breaks existing links to https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/

Reply:
When I build these changes locally, it seemed to still work fine

Reply:
Thanks for double checking